### PR TITLE
add username and password options to docker login

### PR DESCRIPTION
**Pull request by Sabuto**
This enables people to use the username and password for login to docker which sometimes causes errors with uploading.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Sabuto, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-builder#37).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: builder.sh [hunk 1]
```diff
@@ -107,6 +109,10 @@ Options:
        Check if the version already exists before starting the build.
     --docker-login
        Login into docker hub on startup (need '-ti' docker opts)
+    --docker-user
+       Username to login into docker with
+    --docker-pass
```
Comment (pvizeli) [inline builder.sh]: `--docker-password`

Comment (Sabuto) [reply builder.sh]: all updated :)

**Comment by pvizeli** [comment]
I was over the code. Can you remove the old `--docker-login` ? That was never useful like your code. I used this external build servers, but now we have all CI based. Thanks! After that ready to merge

**Comment by Sabuto** [comment]
all done :)

**Pull request merged**